crates/bevy_ui/src/render/mod.rs

```diff
@@ -723,8 +823,24 @@ pub struct UiBatch {
     pub camera: Entity,
 }
 
-const TEXTURED_QUAD: u32 = 0;
-const UNTEXTURED_QUAD: u32 = 1;
+/// The values here should match the values for the constants in `ui.wgsl`
+pub mod shader_flags {
+    pub const UNTEXTURED: u32 = 0;
+    pub const TEXTURED: u32 = 1;
+    pub const BOX_SHADOW: u32 = 2;
+    pub const DISABLE_AA: u32 = 4;
+    pub const CORNERS: [u32; 4] = [
+        // top left
```

Comment:
These values should be moved to doc strings, rather than just comments :)

Reply:
I suppose I should also remove the `BOX_SHADOW` constant here, since it's just an artifact from the original PR anyway.

Reply:
Yep, that works well.